### PR TITLE
Add Crowdin Integration

### LINKED ISSUE #796: Crowdin integration

### PR DESCRIPTION
**Pull request by taooceros**
close #796

### CODE REVIEW

File: Flow.Launcher/Languages/sk.xaml [hunk 1]
```diff
@@ -18,7 +16,7 @@
 
     <!--  Setting General  -->
     <system:String x:Key="flowlauncher_settings">Nastavenia Flow Launchera</system:String>
-    <system:String x:Key="general">Všeobecné</system:String>
+    <system:String x:Key="general">Všeobecn</system:String>
```
Comment (kubalav) [inline Flow.Launcher/Languages/sk.xaml:19]: Why is letter "é" cut off?

**Comment by jjw24** [comment]
before merging into dev, i want to squash all the commits into one. also get the other mr into here too

File: Flow.Launcher/Languages/sk.xaml (hunk 1 above)
Comment (taooceros) [reply Flow.Launcher/Languages/sk.xaml:19]: I think some translation is loaded in a shared memory way? So a bit character seems being cut off? I don't know how crowdin implement that but I do find that chinese translation also get a bit cut off as well.

Comment (taooceros) [reply Flow.Launcher/Languages/sk.xaml:19]: Probably some special character are ignored there....Would you please do a proofread for that?

File: Flow.Launcher/Languages/sk.xaml [hunk 2]
```diff
@@ -18,7 +16,7 @@
 
     <!--  Setting General  -->
     <system:String x:Key="flowlauncher_settings">Nastavenia Flow Launchera</system:String>
-    <system:String x:Key="general">Všeobecné</system:String>
```
Comment (kubalav) [inline Flow.Launcher/Languages/sk.xaml:21]: <system:String x:Key="general">Všeobecné</system:String>

File: Flow.Launcher/Languages/sk.xaml [hunk 3]
```diff
@@ -103,7 +100,7 @@
     <system:String x:Key="shadowEffectCPUUsage">Tieňový efekt významne využíva GPU. Neodporúča sa, ak je výkon počítača obmedzený.</system:String>
     <system:String x:Key="windowWidthSize">Veľkosť šírky okna</system:String>
     <system:String x:Key="useGlyphUI">Použiť ikony Segoe Fluent</system:String>
-    <system:String x:Key="useGlyphUIEffect">Použiť ikony Segoe Fluent, ak sú podporované</system:String>
+    <system:String x:Key="useGlyphUIEffect">Použiť ikony Segoe Fluent, ak sú podporovan</system:String>
```
Comment (kubalav) [inline Flow.Launcher/Languages/sk.xaml:103]:
```suggestion
    <system:String x:Key="useGlyphUIEffect">Použiť ikony Segoe Fluent, ak sú podporovan</system:String>
```

File: Flow.Launcher/Languages/sk.xaml (hunk 1 above)
Comment (kubalav) [inline Flow.Launcher/Languages/sk.xaml:19]:
```suggestion
    <system:String x:Key="general">Všeobecné</system:String>
```

File: Flow.Launcher/Languages/sk.xaml [hunk 4]
```diff
@@ -117,7 +114,7 @@
     <system:String x:Key="serverCantBeEmpty">Pole Server nemôže byť prázdne</system:String>
     <system:String x:Key="portCantBeEmpty">Pole Port nemôže byť prázdne</system:String>
     <system:String x:Key="invalidPortFormat">Neplatný formát portu</system:String>
-    <system:String x:Key="saveProxySuccessfully">Nastavenie proxy úspešne uložené</system:String>
+    <system:String x:Key="saveProxySuccessfully">Nastavenie proxy úspešne uložen</system:String>
```
Comment (kubalav) [inline Flow.Launcher/Languages/sk.xaml:117]:
```suggestion
    <system:String x:Key="saveProxySuccessfully">Nastavenie proxy úspešne uložené</system:String>
```

File: Flow.Launcher/Languages/sk.xaml [hunk 5]
```diff
@@ -159,8 +156,8 @@
     <system:String x:Key="cannotFindSpecifiedPlugin">Nepodarilo sa nájsť zadaný plugin</system:String>
     <system:String x:Key="newActionKeywordsCannotBeEmpty">Nová skratka pre akciu nemôže byť prázdna</system:String>
     <system:String x:Key="newActionKeywordsHasBeenAssigned">Nová skratka pre akciu bola priradená pre iný plugin, prosím, zvoľte inú skratku</system:String>
-    <system:String x:Key="success">Úspešné</system:String>
-    <system:String x:Key="completedSuccessfully">Úspešne dokončené</system:String>
+    <system:String x:Key="success">Úspešn</system:String>
```
Comment (kubalav) [inline Flow.Launcher/Languages/sk.xaml:159]:
```suggestion
    <system:String x:Key="success">Úspešné</system:String>
```

File: Flow.Launcher/Languages/sk.xaml [hunk 6]
```diff
@@ -159,8 +156,8 @@
...
+    <system:String x:Key="completedSuccessfully">Úspešne dokončen</system:String>
```
Comment (kubalav) [inline Flow.Launcher/Languages/sk.xaml:160]:
```suggestion
    <system:String x:Key="completedSuccessfully">Úspešne dokončené</system:String>
```

File: Flow.Launcher/Languages/sk.xaml [hunk 7]
```diff
@@ -180,13 +177,13 @@
     <system:String x:Key="reportWindow_reproduce">Prosím, napíšte nám, ako došlo k pádu aplikácie, aby sme to mohli opraviť</system:String>
     <system:String x:Key="reportWindow_send_report">Odoslať hlásenie</system:String>
     <system:String x:Key="reportWindow_cancel">Zrušiť</system:String>
-    <system:String x:Key="reportWindow_general">Všeobecné</system:String>
+    <system:String x:Key="reportWindow_general">Všeobecn</system:String>
```
Comment (kubalav) [inline Flow.Launcher/Languages/sk.xaml:180]:
```suggestion
    <system:String x:Key="reportWindow_general">Všeobecné</system:String>
```

File: Flow.Launcher/Languages/sk.xaml [hunk 8]
```diff
@@ -180,13 +177,13 @@
...
     <system:String x:Key="reportWindow_exceptions">Výnimky</system:String>
     <system:String x:Key="reportWindow_exception_type">Typ výnimky</system:String>
     <system:String x:Key="reportWindow_source">Zdroj</system:String>
     <system:String x:Key="reportWindow_stack_trace">Stack Trace</system:String>
     <system:String x:Key="reportWindow_sending">Odosiela sa</system:String>
-    <system:String x:Key="reportWindow_report_succeed">Hlásenie bolo úspešne odoslané</system:String>
+    <system:String x:Key="reportWindow_report_succeed">Hlásenie bolo úspešne odoslan</system:String>
```
Comment (kubalav) [inline Flow.Launcher/Languages/sk.xaml:186]:
```suggestion
    <system:String x:Key="reportWindow_report_succeed">Hlásenie bolo úspešne odoslané</system:String>
```

File: Flow.Launcher/Languages/sk.xaml [hunk 9]
```diff
@@ -210,7 +207,7 @@
     <system:String x:Key="update_flowlauncher_fail">Aktualizácia zlyhala</system:String>
     <system:String x:Key="update_flowlauncher_check_connection">Skontrolujte pripojenie a skúste aktualizovať nastavenia servera proxy na github-cloud.s3.amazonaws.com.</system:String>
     <system:String x:Key="update_flowlauncher_update_restart_flowlauncher_tip">Tento upgrade reštartuje Flow Launcher</system:String>
-    <system:String x:Key="update_flowlauncher_update_upadte_files">Nasledujúce súbory budú aktualizované</system:String>
+    <system:String x:Key="update_flowlauncher_update_upadte_files">Nasledujúce súbory budú aktualizovan</system:String>
```
Comment (kubalav) [inline Flow.Launcher/Languages/sk.xaml:210]:
```suggestion
    <system:String x:Key="update_flowlauncher_update_upadte_files">Nasledujúce súbory budú aktualizované</system:String>
```

File: Plugins/Flow.Launcher.Plugin.Explorer/Languages/sk.xaml [hunk 10]
```diff
@@ -1,66 +1,67 @@
...
\ No newline at end of file
+﻿<?xml version="1.0"?>
+<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:system="clr-namespace:System;assembly=mscorlib">
+
+    <!--Dialogues-->
+    <system:String x:Key="plugin_explorer_make_selection_warning">Najprv vyberte položku</system:String>
+    <system:String x:Key="plugin_explorer_select_folder_link_warning">Vyberte odkaz na priečinok</system:String>
+    <system:String x:Key="plugin_explorer_delete_folder_link">Naozaj chcete odstrániť {0}?</system:String>
+    <system:String x:Key="plugin_explorer_deletefilefolderconfirm">Naozaj chcete natrvalo odstrániť túto položku {0}?</system:String>
+    <system:String x:Key="plugin_explorer_deletefilefoldersuccess">Odstránenie bolo úspešn</system:String>
```
Comment (kubalav) [inline Plugins/Flow.Launcher.Plugin.Explorer/Languages/sk.xaml:9]:
```suggestion
    <system:String x:Key="plugin_explorer_deletefilefoldersuccess">Odstránenie bolo úspešné</system:String>

```

File: Plugins/Flow.Launcher.Plugin.Explorer/Languages/sk.xaml [hunk 11]
```diff
@@ -1,66 +1,67 @@
...
+    <system:String x:Key="plugin_explorer_indexsearchexcludedpaths_header">Vylúčené umiestnenia indexovania</system:String>
+    <system:String x:Key="plugin_explorer_manageindexoptions">Možnosti indexovania</system:String>
+    <system:String x:Key="plugin_explorer_actionkeywordview_search">Vyhľadávanie:</system:String>
+    <system:String x:Key="plugin_explorer_actionkeywordview_pathsearch">Vyhľadávanie umiestnenia:</system:String>
+    <system:String x:Key="plugin_explorer_actionkeywordview_filecontentsearch">Vyhľadávanie obsahu súborov:</system:String>
+    <system:String x:Key="plugin_explorer_actionkeywordview_indexsearch">Vyhľadávanie v indexe:</system:String>
+    <system:String x:Key="plugin_explorer_actionkeywordview_quickaccess">Quick Access:</system:String>
+    <system:String x:Key="plugin_explorer_actionkeyword_current">Aktuálna skratka:</system:String>
+    <system:String x:Key="plugin_explorer_actionkeyword_done">Hotovo</system:String>
+    <system:String x:Key="plugin_explorer_actionkeyword_enabled">Povolen</system:String>
```
Comment (kubalav) [inline Plugins/Flow.Launcher.Plugin.Explorer/Languages/sk.xaml:33]:
```suggestion
    <system:String x:Key="plugin_explorer_actionkeyword_enabled">Povolené</system:String>

```

File: Plugins/Flow.Launcher.Plugin.Explorer/Languages/sk.xaml [hunk 12]
```diff
@@ -1,66 +1,67 @@
...
+    <!--Plugin Infos-->
+    <system:String x:Key="plugin_explorer_plugin_name">Explorer</system:String>
+    <system:String x:Key="plugin_explorer_plugin_description">Vyhľadáva a spravuje súbory a priečinky. Explorer používa indexovanie vyhľadávania vo Windowse</system:String>
+
+    <!--Context menu items-->
+    <system:String x:Key="plugin_explorer_copypath">Kopírovať cestu</system:String>
+    <system:String x:Key="plugin_explorer_copyfilefolder">Kopírovať</system:String>
+    <system:String x:Key="plugin_explorer_deletefilefolder">Odstrániť</system:String>
+    <system:String x:Key="plugin_explorer_path">Umiestnenie:</system:String>
+    <system:String x:Key="plugin_explorer_deletefilefolder_subtitle">Odstrániť vybran</system:String>
```
Comment (kubalav) [inline Plugins/Flow.Launcher.Plugin.Explorer/Languages/sk.xaml:45]:
```suggestion
    <system:String x:Key="plugin_explorer_deletefilefolder_subtitle">Odstrániť vybrané</system:String>

```

File: Plugins/Flow.Launcher.Plugin.Explorer/Languages/sk.xaml [hunk 13]
```diff
@@ -1,66 +1,67 @@
...
+    <system:String x:Key="plugin_explorer_openwitheditor">Otvoriť editorom:</system:String>
+    <system:String x:Key="plugin_explorer_excludefromindexsearch">Vylúčiť položku a jej podpriečinky z indexu vyhľadávania</system:String>
+    <system:String x:Key="plugin_explorer_excludedfromindexsearch_msg">Vylúčiť z indexu vyhľadávania</system:String>
+    <system:String x:Key="plugin_explorer_openindexingoptions">Otvoriť možnosti vyhľadávania vo Windowse</system:String>
+    <system:String x:Key="plugin_explorer_openindexingoptions_subtitle">Správa indexovaných súborov a priečinkov</system:String>
+    <system:String x:Key="plugin_explorer_openindexingoptions_errormsg">Nepodarilo sa otvoriť možnosti indexu vyhľadávania</system:String>
+    <system:String x:Key="plugin_explorer_add_to_quickaccess_title">Pridať do Rýchleho prístupu</system:String>
+    <system:String x:Key="plugin_explorer_add_to_quickaccess_subtitle">Pridať tento {0} do Rýchleho prístupu</system:String>
+    <system:String x:Key="plugin_explorer_addfilefoldersuccess">Úspešne pridan</system:String>
```
Comment (kubalav) [inline Plugins/Flow.Launcher.Plugin.Explorer/Languages/sk.xaml:58]:
```suggestion
    <system:String x:Key="plugin_explorer_addfilefoldersuccess">Úspešne pridané</system:String>

```

File: Plugins/Flow.Launcher.Plugin.Explorer/Languages/sk.xaml [hunk 14]
```diff
@@ -1,66 +1,67 @@
...
+    <system:String x:Key="plugin_explorer_addfilefoldersuccess_detail">Úspešne pridané do Rýchleho prístupu</system:String>
+    <system:String x:Key="plugin_explorer_removefilefoldersuccess">Úspešne odstránen</system:String>
```
Comment (kubalav) [inline Plugins/Flow.Launcher.Plugin.Explorer/Languages/sk.xaml:60]:
```suggestion
    <system:String x:Key="plugin_explorer_removefilefoldersuccess">Úspešne odstránené</system:String>

```

File: Plugins/Flow.Launcher.Plugin.PluginsManager/Languages/sk.xaml [hunk 15]
```diff
@@ -1,41 +1,40 @@
...
-    
-</ResourceDictionary>
\ No newline at end of file
+﻿<?xml version="1.0"?>
+<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:system="clr-namespace:System;assembly=mscorlib">
+
+    <!--Dialogues-->
+    <system:String x:Key="plugin_pluginsmanager_downloading_plugin">Sťahovanie pluginu</system:String>
+    <system:String x:Key="plugin_pluginsmanager_please_wait">Čakajte, prosím…</system:String>
+    <system:String x:Key="plugin_pluginsmanager_download_success">Úspešne stiahnut</system:String>
```
Comment (kubalav) [inline Plugins/Flow.Launcher.Plugin.PluginsManager/Languages/sk.xaml:7]:
```suggestion
    <system:String x:Key="plugin_pluginsmanager_download_success">Úspešne stiahnuté</system:String>

```

File: Plugins/Flow.Launcher.Plugin.PluginsManager/Languages/sk.xaml [hunk 16]
```diff
@@ -1,41 +1,40 @@
...
+    <system:String x:Key="plugin_pluginsmanager_uninstall_prompt">{0} od {1} {2}{3}Chcete odinštalovať tento plugin? Po odinštalovaní sa Flow automaticky reštartuje.</system:String>
+    <system:String x:Key="plugin_pluginsmanager_install_prompt">{0} by {1} {2}{3}Chcete nainštalovať tento plugin? Po nainštalovaní sa Flow automaticky reštartuje.</system:String>
+    <system:String x:Key="plugin_pluginsmanager_install_title">Inštalovať plugin</system:String>
+    <system:String x:Key="plugin_pluginsmanager_uninstall_title">Odinštalovať plugin</system:String>
+    <system:String x:Key="plugin_pluginsmanager_install_errormetadatafile">Inštalácia zlyhala: nepodarilo sa nájsť metadáta súboru plugin.json nového pluginu</system:String>
+    <system:String x:Key="plugin_pluginsmanager_install_error_title">Chyba inštalácie pluginu</system:String>
+    <system:String x:Key="plugin_pluginsmanager_install_error_subtitle">Nastala chyba počas inštaláciu pluginu {0}</system:String>
```
Comment (kubalav) [inline Plugins/Flow.Launcher.Plugin.PluginsManager/Languages/sk.xaml:14]:
```suggestion
    <system:String x:Key="plugin_pluginsmanager_install_error_subtitle">Nastala chyba počas inštalácie pluginu {0}</system:String>

```

File: Plugins/Flow.Launcher.Plugin.Program/Languages/sk.xaml [hunk 17]
```diff
@@ -1,58 +1,57 @@
...
+    <system:String x:Key="flowlauncher_plugin_program_file_suffixes">Prípony súboru:</system:String>
+    <system:String x:Key="flowlauncher_plugin_program_max_search_depth">Max. hĺbka hľadania (-1 je neobm.):</system:String>
+
+    <system:String x:Key="flowlauncher_plugin_program_pls_select_program_source">Prosím, zadajte zdroj programu</system:String>
+    <system:String x:Key="flowlauncher_plugin_program_delete_program_source">Naozaj chcete odstrániť vybrané zdroje programov?</system:String>
+
+    <system:String x:Key="flowlauncher_plugin_program_update">Aktualizovať</system:String>
+    <system:String x:Key="flowlauncher_plugin_program_only_index_tip">Flow Launcher bude indexovať iba súbory s nasledujúcimi príponami:</system:String>
+    <system:String x:Key="flowlauncher_plugin_program_split_by_tip">(Každú príponu oddeľte ;)</system:String>
+    <system:String x:Key="flowlauncher_plugin_program_update_file_suffixes">Prípony boli úspešne aktualizovan</system:String>
```
Comment (kubalav) [inline Plugins/Flow.Launcher.Plugin.Program/Languages/sk.xaml:34]:
```suggestion
    <system:String x:Key="flowlauncher_plugin_program_update_file_suffixes">Prípony boli úspešne aktualizované</system:String>

```

File: Plugins/Flow.Launcher.Plugin.Program/Languages/sk.xaml [hunk 18]
```diff
@@ -1,58 +1,57 @@
...
+
+    <system:String x:Key="flowlauncher_plugin_program_invalid_path">Neplatná cesta</system:String>
+
+    <system:String x:Key="flowlauncher_plugin_program_customizedexplorer">Vlastný správca súborov</system:String>
+    <system:String x:Key="flowlauncher_plugin_program_args">Arg.</system:String>
+    <system:String x:Key="flowlauncher_plugin_program_tooltip_customizedexplorer">Môžete si prispôsobiť otváranie umiestnenia priečinka vložením Premenných prostredia, ktoré chcete použiť. Dostupnosť premenných prostredia môžete vyskúšať cez príkazový riadok.</system:String>
+    <system:String x:Key="flowlauncher_plugin_program_tooltip_args">Zadajte argumenty, ktoré chcete pridať pre správcu súborov. %s pre rodičovský priečinok, %f pre celú cestu (funguje iba pre win32). Pre podrobnosti pozrite webovú stránku správcu súborov.</system:String>
+
+    <!--Dialogs-->
+    <system:String x:Key="flowlauncher_plugin_program_disable_dlgtitle_success">Úspešn</system:String>
```
Comment (kubalav) [inline Plugins/Flow.Launcher.Plugin.Program/Languages/sk.xaml:53]:
```suggestion
    <system:String x:Key="flowlauncher_plugin_program_disable_dlgtitle_success">Úspešné</system:String>

```

File: Plugins/Flow.Launcher.Plugin.Sys/Languages/sk.xaml [hunk 19]
```diff
@@ -1,38 +1,37 @@
...
+    <system:String x:Key="flowlauncher_plugin_sys_save_all_settings">Uložiť všetky nastavenia Flow Launchera</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_reload_plugin_data">Aktualizovať všetky dáta pluginov od spustenia Flow Launchera. Pluginy musia túto funkciu podporovať.</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_open_log_location">Otvoriť umiestnenie denníka Flow Launchera</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_check_for_update">Skontrolovať aktualizácie Flow Launchera</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_open_docs_tips">V dokumentácii k aplikácii Flow Launcher nájdete ďalšiu pomoc a tipy na používanie</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_open_userdata_location">Otvoriť umiestnenie, kde sú uložené nastavenia Flow Launchera</system:String>
+
+    <!--Dialogs-->
+    <system:String x:Key="flowlauncher_plugin_sys_dlgtitle_success">Úspešn</system:String>
+    <system:String x:Key="flowlauncher_plugin_sys_dlgtext_all_settings_saved">Všetky nastavenia Flow Launchera uložen</system:String>
```
Comment (kubalav) [inline Plugins/Flow.Launcher.Plugin.Sys/Languages/sk.xaml:28]:
```suggestion
    <system:String x:Key="flowlauncher_plugin_sys_dlgtext_all_settings_saved">Všetky nastavenia Flow Launchera uložené</system:String>

```

File: Plugins/Flow.Launcher.Plugin.Sys/Languages/sk.xaml [hunk 20]
```diff
@@ -1,38 +1,37 @@
...
+    <system:String x:Key="flowlauncher_plugin_sys_dlgtext_all_applicableplugins_reloaded">Všetky dáta pluginov aktualizovan</system:String>
```
Comment (kubalav) [inline Plugins/Flow.Launcher.Plugin.Sys/Languages/sk.xaml:29]:
```suggestion
    <system:String x:Key="flowlauncher_plugin_sys_dlgtext_all_applicableplugins_reloaded">Všetky dáta pluginov aktualizované</system:String>

```

File: Plugins/Flow.Launcher.Plugin.WebSearch/Languages/sk.xaml [hunk 21]
```diff
@@ -1,38 +1,43 @@
...
+    <system:String x:Key="flowlauncher_plugin_websearch_enable">Povoliť</system:String>
+    <system:String x:Key="flowlauncher_plugin_websearch_select_icon">Vybrať ikonu</system:String>
+    <system:String x:Key="flowlauncher_plugin_websearch_icon">Ikona</system:String>
+    <system:String x:Key="flowlauncher_plugin_websearch_cancel">Zrušiť</system:String>
+    <system:String x:Key="flowlauncher_plugin_websearch_invalid_web_search">Neplatné webové vyhľadávanie</system:String>
+    <system:String x:Key="flowlauncher_plugin_websearch_input_title">Zadajte názov</system:String>
+    <system:String x:Key="flowlauncher_plugin_websearch_input_action_keyword">Zadajte skratku akcie</system:String>
+    <system:String x:Key="flowlauncher_plugin_websearch_input_url">Zadajte URL</system:String>
+    <system:String x:Key="flowlauncher_plugin_websearch_action_keyword_exist">Zadaná skratka akcie už existuje, zadajte inú</system:String>
+    <system:String x:Key="flowlauncher_plugin_websearch_succeed">Úspešn</system:String>
```
Comment (kubalav) [inline Plugins/Flow.Launcher.Plugin.WebSearch/Languages/sk.xaml:37]:
```suggestion
    <system:String x:Key="flowlauncher_plugin_websearch_succeed">Úspešné</system:String>

```

File: Flow.Launcher/Languages/sk.xaml (hunk 1 above)
Comment (kubalav) [reply Flow.Launcher/Languages/sk.xaml]: All occurrences fixed!
I found that it's occurred if character "é" is followed by "<" only.

Comment (taooceros) [reply Flow.Launcher/Languages/sk.xaml]: Cool! I will sync it to crowdin

**Comment by taooceros** [comment]
I will contact crowdin for the translation upload issue.

File: crowdin.yml [hunk 22]
```diff
@@ -0,0 +1,6 @@
+files:
+  - source: Languages/en.xaml
+    translation: /%original_path%/%two_letters_code%.xaml
+  - source: Flow.Launcher.Plugin.WindowsSettings/Properties/Resources.resx
```
Comment (taooceros) [inline crowdin.yml:4]:
```suggestion
  - source: Properties/Resources.resx
```

**Comment by jjw24** [comment]
@taooceros @kubalav this should be good to merge into dev right? (let me merge it if it is)

File: Flow.Launcher/Languages/sk.xaml [hunk 23]
```diff
@@ -1,217 +1,232 @@
...
+    <system:String x:Key="delete">Odstrániť</system:String>
+    <system:String x:Key="edit">Upraviť</system:String>
+    <system:String x:Key="add">Pridať</system:String>
+    <system:String x:Key="pleaseSelectAnItem">Vyberte položku, prosím</system:String>
+    <system:String x:Key="deleteCustomHotkeyWarning">Ste si istý, že chcete odstrániť klávesovú skratku {0} pre plugin?</system:String>
+    <system:String x:Key="queryWindowShadowEffect">Tieňový efekt v poli vyhľadávania</system:String>
+    <system:String x:Key="shadowEffectCPUUsage">Tieňový efekt významne využíva GPU. Neodporúča sa, ak je výkon počítača obmedzený.</system:String>
+    <system:String x:Key="windowWidthSize">Veľkosť šírky okna</system:String>
+    <system:String x:Key="useGlyphUI">Použiť ikony Segoe Fluent</system:String>
+    <system:String x:Key="useGlyphUIEffect">Použiť ikony Segoe Fluent, ak sú podporovan</system:String>
```
Comment (kubalav) [inline Flow.Launcher/Languages/sk.xaml:116]:
```suggestion
    <system:String x:Key="useGlyphUIEffect">Použiť ikony Segoe Fluent, ak sú podporované</system:String>

```

File: Flow.Launcher/Languages/sk.xaml [hunk 24]
```diff
@@ -1,217 +1,232 @@
...
+    <!--  Action Keyword Setting Dialog  -->
+    <system:String x:Key="oldActionKeywords">Stará skratka akcie</system:String>
+    <system:String x:Key="newActionKeywords">Nová skratka akcie</system:String>
+    <system:String x:Key="cancel">Zrušiť</system:String>
+    <system:String x:Key="done">Hotovo</system:String>
+    <system:String x:Key="cannotFindSpecifiedPlugin">Nepodarilo sa nájsť zadaný plugin</system:String>
+    <system:String x:Key="newActionKeywordsCannotBeEmpty">Nová skratka pre akciu nemôže byť prázdna</system:String>
+    <system:String x:Key="newActionKeywordsHasBeenAssigned">Nová skratka pre akciu bola priradená pre iný plugin, prosím, zvoľte inú skratku</system:String>
+    <system:String x:Key="success">Úspešné</system:String>
+    <system:String x:Key="completedSuccessfully">Úspešne dokončen</system:String>
```
Comment (kubalav) [inline Flow.Launcher/Languages/sk.xaml:178]:
```suggestion
    <system:String x:Key="completedSuccessfully">Úspešne dokončené</system:String>

```

File: Flow.Launcher/Languages/sk.xaml [hunk 25]
```diff
@@ -1,217 +1,232 @@
...
+    <system:String x:Key="reportWindow_reproduce">Prosím, napíšte nám, ako došlo k pádu aplikácie, aby sme to mohli opraviť</system:String>
+    <system:String x:Key="reportWindow_send_report">Odoslať hlásenie</system:String>
+    <system:String x:Key="reportWindow_cancel">Zrušiť</system:String>
+    <system:String x:Key="reportWindow_general">Všeobecné</system:String>
+    <system:String x:Key="reportWindow_exceptions">Výnimky</system:String>
+    <system:String x:Key="reportWindow_exception_type">Typ výnimky</system:String>
+    <system:String x:Key="reportWindow_source">Zdroj</system:String>
+    <system:String x:Key="reportWindow_stack_trace">Trasovanie zásobníka</system:String>
+    <system:String x:Key="reportWindow_sending">Odosiela sa</system:String>
+    <system:String x:Key="reportWindow_report_succeed">Hlásenie bolo úspešne odoslan</system:String>
```
Comment (kubalav) [inline Flow.Launcher/Languages/sk.xaml:204]:
```suggestion
    <system:String x:Key="reportWindow_report_succeed">Hlásenie bolo úspešne odoslané</system:String>

```

**Comment by jjw24** [comment]
hey @kubalav how come your changes are not via Crowdin?

**Comment by kubalav** [comment]
Edited and ready to merge.

**Comment by kubalav** [comment]
> hey @kubalav how come your changes are not via Crowdin?

I checked diff in Github and I didn't realize to make changes in Crowdin. Next time.

**Comment by jjw24** [comment]
no probs, was just curious

**Review by jjw24 — APPROVED** [review]

**Pull request merged**